packages/widget-chat/src/components/activity-title-bar/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';
+
+import ActivityTitleBar from '.';
+
+let user;
+let component;
+
+describe(`ActivityTitleBar component`, () => {
```

Comment:
ok, yea, we may have some naming issues in the web client code base. let's get things right here. :)